### DISCUSSION TITLE
作者您好,我想提两个功能建议

### DISCUSSION THREAD

**Discussion by xiaowu-li** [zhongyang219/MusicPlayer2#654] [Ideas]
一、下载歌曲后把文件放到播放文件后,播放列表不会自动刷新,选项设置里的媒体库页面有一个刷新媒体库按钮不知道是不是可以刷新播放列表,如果可以的话吧这个刷新播放列表的功能放在播放列表页面会更好。
二、在各处下载的歌曲名字会很混乱，我发现在播放页面的CTRL+N曲目信息页面的标签信息可以自定义，然后播放列表的名字会发生变化，所以我的想法是否可以吧这个![建议](https://github.com/zhongyang219/MusicPlayer2/assets/61909237/d046aef6-2036-45e6-9b37-ca987764ccef)

**Comment by xiaowu-li** [comment]
是否可以吧文件名哪一项变成可自定义，并且会同步到文件夹顺便修改文件夹里的文件名字

**Comment by lrisora** [comment]
关于1，播放列表没办法自动从文件夹同步歌曲，
因为播放列表不在乎文件来自哪个文件夹，也就是支持添加来自不同文件夹的歌曲，这与自动从文件夹同步冲突
你可以使用文件夹模式，不使用播放列表模式，文件夹模式总是与文件夹同步的状态（参考F1的帮助）

关于2，可以先修改好歌曲元数据（标签）之后在播放列表中多选歌曲之后右键选择”重命名“命令
应该可以满足简单的需要

![圖片](https://github.com/zhongyang219/MusicPlayer2/assets/43085437/65a78b58-55e7-48bb-8d64-87c022ea32ef)

**Reply by lrisora** [reply]
等等，我突然意识到重命名功能的一个问题，
这个功能本应仅使用第一个艺术家的，现在会使用所有艺术家，
这可能会打爆windows的路径长度限制
